deployment/kubernetes/generic/bookie.yaml

```diff
@@ -23,10 +23,10 @@ kind: ConfigMap
 metadata:
     name: bookie-config
 data:
-    PULSAR_MEM: "\" -Xms512m -Xmx512m -XX:MaxDirectMemorySize=1g\""
-    dbStorage_writeCacheMaxSizeMb: "256" # Write cache size (direct memory)
-    dbStorage_readAheadCacheMaxSizeMb: "256" # Read cache size (direct memory)
-    zkServers: zk-0.zookeeper,zk-1.zookeeper,zk-2.zookeeper
+    PULSAR_MEM: "\" -Xms64m -Xmx256m -XX:MaxDirectMemorySize=256m\""
```

Comment:
The "generic" instructions were also meant for a "custom" K8S cluster, though not necessarily a minikube one. Should we rather differentiate the 2 cases?

Reply:
the changes are I made here are reducing the memory usage for people to start. so they are still valid for a "custom" K8S cluster, we don't need to differentiate them. because in general, for a "custom" K8S cluster, we don't know exactly the resources people have, so starting from smaller values would be better for most people.